### PR TITLE
Update the PostgreSQL version to latest available in the 9.5 series

### PR DESCRIPTION
**Pull request by justinclift**
## What type of PR is this? (check all applicable)

- [x] Security update

In theory (!), this updated PG version should require no corresponding changes made to anything else.

Updating to a more modern PG release would have benefits ([parallel query](https://www.postgresql.org/docs/11/parallel-query.html) stands out), but may require some code/SQL changes to match.  So, keeping this simple for now. :smile:

### CODE REVIEW

File: setup/docker-compose.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ services:
     image: redis:3.0-alpine
     restart: always
   postgres:
-    image: postgres:9.5.6-alpine
+    image: postgres:9.5.16-alpine
```
Comment (arikfr) [inline setup/docker-compose.yml]: Does Postgres publish `postgres:9.5` image that tracks latest patch version?

Comment (justinclift) [reply setup/docker-compose.yml]: Good point.  Yes it does.  1 sec, will update. :smile:

**Comment by justinclift** [comment]
As a data point, Redash seems to work happily with the PostgreSQL `11-alpine` image as well.

Testing it here now (only lightly though).

**Comment by arikfr** [comment]
Thanks and thanks for testing w/ Pg11!

**Pull request merged**